<launch>

	<arg name="imu_signal"		    default="/imu/data"/>
	<arg name="imu_filtered"		default="/imu/filtered"/>
	<arg name="filter_order"		default="4"/> 
	<arg name="filter_fc"			default="45"/> 
	<arg name="filter_fs"			default="100"/> 
	<arg name="content"			    default="linear_acceleration"/> 
    <!-- 0x28 == 40 is the default for BNO055 -->

    <node ns="imu" name="imu_filter" pkg="imu_bno055" type="low_pass_filter.py" respawn="true" respawn_delay="2">
        <param name="imu_signal"    type="string"   value="$(arg imu_signal)"/> 
        <param name="imu_filtered"  type="string"   value="$(arg imu_filtered)"/> 
        <param name="filter_order"  type="int"      value="$(arg filter_order)"/> 
        <param name="filter_fc"     type="double"    value="$(arg filter_fc)"/> 
        <param name="filter_fs"     type="double"    value="$(arg filter_fs)"/> 
        <param name="content"       type="string"   value="$(arg content)"/> 
    </node>
</launch>
